index.html

```diff
@@ -43,9 +43,146 @@ <h1>Hey there. I’m Salim.<br><span class="highlight">I’m a software develope
                 <li><a href="#"><img src="./static/images/Icons/medium.png" alt="medium-image"></a></li>
             </ul>
             <div class="action">
-                <button><img src="./static/images/Buttons/down_arrow.png" alt="down-arrow-image"></button>
+                <button type="button"><img src="./static/images/Buttons/down_arrow.png" alt="down-arrow-image"></button>
             </div>
         </section>
+
+        <section id="works">
+            <div class="primary-text">
+                <h2>My Recent Works</h2>
+            </div>
+
+            <ul class="projects">
+                <li class="project-card">
+                    <img src="./static/images/Projects/img-placeholder.png" alt="project-image">
+                    <p class="title">Multi-Post Stories <br> Gain+Glory</p>
```

Comment:
- [ ] great job on the structure of your code 👍 according to the requirements however this title should be a header tag `<hx>` instead of a paragraph `<p>`